packages/berlin/src/pages/Holding.tsx

```diff
@@ -0,0 +1,47 @@
+// React and third-party libraries
+import { useQuery } from '@tanstack/react-query';
+
+// Components
+import { FlexColumn } from '../components/containers/FlexColum.styled';
+import { Body } from '../components/typography/Body.styled';
+import Link from '../components/link';
+import { useNavigate } from 'react-router-dom'; // Import useNavigate
+
+// API Calls
+import { fetchEvents } from 'api';
+
+function Holding() {
+  const navigate = useNavigate();
+
+  const { data: events, isLoading: eventsLoading } = useQuery({
+    queryKey: ['events'],
+    queryFn: fetchEvents,
+  });
+
+  if (eventsLoading) {
+    return <Body>Loading events...</Body>;
+  }
+
+  // Define onClick handler to navigate to registration page and trigger reload
+  const handleRegistrationClick = () => {
+    navigate(`/events/${events?.[0]?.id}/register`, { replace: true }); 
```

Comment:
We can get the `eventId` from the url by using react-router `useParams` hook, that would remove the need of doing the `events` query here. Also, the `replace` replaces the history navigation, so user won't be able to go back, not sure if thats what we want, as you'll face this scenario:

https://github.com/lexicongovernance/forum-frontend/assets/59750365/e9862619-27b4-401f-be77-cc5c57a04da8

Reply:
Done.